.readthedocs.yaml

```
@@ -0,0 +1,16 @@
+version: 2
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)